File: SMS Scripts/send_sms.py
from twilio.rest import TwilioRestClient
 # Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "AC32a3c49700934481addd5ce1659f04d2"
auth_token  = "45f2016a23a6629ecc9df46ad2f3212a"
client = TwilioRestClient(account_sid, auth_token)
 
message = client.messages.create(body="Twilio SMS for HackWeek",
    to="+16477450709",    # Replace with your phone number
    from_="+12264009012") # Replace with your Twilio number
print message.sid

File: SMS Scripts/send-sms.rb
require 'rubygems'
require 'twilio-ruby'
 
account_sid = "AC164a87fb536f9ead23575f129196c639"
auth_token = "45f2016a23a6629ecc9df46ad2f3212a"
client = Twilio::REST::Client.new account_sid, auth_token
 
from = "+12264003340" # Your Twilio number
 
friends = {
"+16477450709" => "Alex",
"+16477467048" => "Josh"
}
friends.each do |key, value|
  client.account.messages.create(
    :from => from,
    :to => key,
    :body => "Hey #{value}, Monkey party at 6PM. Bring Bananas!"
  )
  puts "Sent message to #{value}"
end
